reversers/database-synchroniser/plugins/liquibase/.gitignore

```diff
@@ -0,0 +1 @@
+/jars/
```

Comment:
Oh well ! I'm surprised to see this as the jars actually are committed in the git repository. Maybe the only action required would be to commit a removal of the jars?

Reply:
Never mind. My mistake probably. The jars are not here :/